tests/cli/workflow/balsamic/test_store_housekeeper.py

```diff
@@ -126,3 +126,5 @@ def test_valid_case_already_added(
     assert result.exit_code != EXIT_SUCCESS
     # THEN user should be informed that bundle was already added
     assert "Bundle already added" in caplog.text
+    # THEN statusdb action should be set ot hold
+    assert balsamic_context["BalsamicAnalysisAPI"].store.family(case_id).action == "hold"
```

Comment:
This action is by "tradition" used for holding the analysis and "Prodteam" will probably be confused if we use it in new ways